build.gradle

```diff
@@ -30,39 +31,59 @@ repositories {
 
 
 apply plugin: 'org.junit.platform.gradle.plugin'
-//apply plugin: 'kotlin'
+apply plugin: 'idea'
 
 group = 'com.buildit'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = 1.8
 
+sourceSets {
+    'test-e2e' {
+        kotlin {
+            srcDir files('src/test-e2e/kotlin')
+        }
+        resources.srcDir files('src/test-e2e/resources')
+    }
+}
+
+idea {
+    module {
+        testSourceDirs += files('src/test-e2e/kotlin')
+    }
+}
```

Comment:
This is my attempt to get IntelliJ to recognize this folder as a "test sources" folder.  It doesn't work so we could get rid of it